index.html

```diff
@@ -12,291 +12,265 @@
 </head>
 <body>
     <header>
-        <div class="wrapper">
             <nav>
```

Comment:
Remove all white spaces from code base

Reply:
@devalibello still having problem understanding the changes